flink-runtime/src/main/java/org/apache/flink/runtime/io/disk/iomanager/IOManager.java

```diff
@@ -242,7 +242,7 @@ public abstract BulkBlockChannelReader createBulkBlockChannelReader(
 	/**
 	 * The manager used for creating/deleting file channels based on config temp dirs.
 	 */
-	private static class FileChannelManager {
+	public static class FileChannelManager {
```

Comment:
I would move it into a separate class because it does not depend on `IOManager` and now also used completely independently from `IOManager`.

Reply:
I also considered it before. Because the `IOManager`  relies on `FileChannelManager` to implement some public methods, so I keep `FileChannelManager` inside `IOManager`. It also makes sense to move it outside.